ipi/inputs/simulation.py

```diff
@@ -153,6 +154,10 @@ def store(self, simul):
                     _iobj = iforcefields.InputFFLennardJones()
                     _iobj.store(_obj)
                     self.extra[_ii] = ("fflj", _iobj)
+                elif isinstance(_obj, eforcefields.FFQUIP):
```

Comment:
This might also need some work once you merge with feat/fast_ff as I added some provisions to reuse InputXXX objects when storing the checkpoint file (saving like 0.5s/step in complex REMD simulations)